terminaltables.py

```diff
@@ -22,21 +22,56 @@
 import re
 import struct
 import sys
+import unicodedata
+
+PY2 = sys.version_info[0] == 2
 
 try:
     import fcntl
     import termios
-except ImportError:
+except ImportError:  # pragma: no cover
     fcntl = None
     termios = None
 
-from colorclass import _WindowsCSBI
+from colorclass import _WindowsCSBI, Color
 
 __author__ = '@Robpol86'
 __license__ = 'MIT'
 __version__ = '1.2.0'
 
 
+if PY2:  # pragma: no cover
+    def _to_unicode(string):
```

Comment:
I also don't like this implementation. _to_unicode() is only used in _get_width(). Can you move this block of code into that function?